index.html

```diff
@@ -31,12 +31,32 @@ <h1 class="greeting h-text">I'm Alexandra <br> Glad to see <br> you!</h1>
                 </p>
                 <span class="light-purple letsconnect">LET'S CONNECT</span>
                 <ul class="icons">
-                <li><img src="twitter.png" alt="icon"></li>
-                <li><img src="Linkedin icon.png" alt="icon"></li>
-                <li><img src="medium.png" alt="icon"></li>
-                <li><img src="github.png" alt="icon"></li>
-                <li><img src="angellist.png" alt="icon"></li>
-                </ul>
+                        <li>
+                          <a href="YOUR SOCIAL MEDIA HANDLE">
```

Comment:
[Optional] By _your social media handle_ the previous reviewer meant that you should use the actual URL of your social media handle, e.g `https://github.com/Naledi-Dikgale`. This is so that they can be navigated to.